project/cmake/addons/CMakeLists.txt

```diff
@@ -275,6 +278,9 @@ foreach(addon ${addons})
         else()
           message(FATAL_ERROR "${id}: invalid or missing addon source directory at ${SOURCE_DIR}")
         endif()
+      else()
+        # add a dummy target for addons that are unsupported on this platform
+        add_custom_target(${id} COMMAND ${CMAKE_COMMAND} -E echo "IGNORED ${id} - not supported on ${CORE_SYSTEM_NAME}\n")
```

Comment:
i mean this alone would have solved the issue without adaptions anywere else ...